docs/_posts/murat-gunay/2022-04-11-spellcheck_clinical_en_3_0.md

```diff
@@ -131,4 +129,4 @@ val assembler = new DocumentAssembler()
 
 ## References
 
-MTSamples, i2b2 clinical notes, and several specific medical corpuses.
```

Comment:
Just a comment - the plural of corpus is `copora`, not corpuses. But no need to change it.

Reply:
I updated the "corpora"  term also in description section.